packages/js/components/src/date-time-picker-control/date-time-picker-control.tsx

```diff
@@ -77,9 +77,7 @@ export const DateTimePickerControl: React.FC< DateTimePickerControlProps > = ( {
 	} );
 
 	const [ inputString, setInputString ] = useState( '' );
-	const [ lastValidDate, setLastValidDate ] = useState< Moment | null >(
-		null
-	);
+	const lastValidDate = useRef< Moment | null >( null );
```

Comment:
Just a heads up that we do have a `usePrevious` hook [here](https://github.com/woocommerce/woocommerce/blob/c2473da79be9333fa0c3f0a3e24008c429107ba3/plugins/woocommerce-admin/client/layout/controller.js#L288-L294) that could be useful if this was stored as state.

But you may be wanting to avoid using state entirely for this piece of data.

Reply:
I ended up reworking things to not need `usePrevious` or `useRef`.